test/extensions/stats_sinks/common/statsd/statsd_test.cc

```diff
@@ -95,13 +95,50 @@ TEST_F(TcpStatsdSinkTest, BasicFlow) {
 
   NiceMock<Stats::MockHistogram> timer;
   timer.name_ = "test_timer";
+  timer.unit_ = Stats::Histogram::Unit::Unspecified;
   EXPECT_CALL(*connection_, write(BufferStringEqual("envoy.test_timer:5|ms\n"), _));
   sink_->onHistogramComplete(timer, 5);
 
   EXPECT_CALL(*connection_, close(Network::ConnectionCloseType::NoFlush));
   tls_.shutdownThread();
 }
 
+TEST_F(TcpStatsdSinkTest, SiSuffix) {
+  InSequence s;
+  expectCreateConnection();
+
+  NiceMock<Stats::MockHistogram> items;
+  items.name_ = "items";
+  items.unit_ = Stats::Histogram::Unit::Unspecified;
```

Comment:
ditto

Reply:
Dropping it from everywhere but `SiSuffix` tests (3 of these total) since this is the whole point of the test to check whether the unit affects the suffix in the expected way and better to have it explicit in the test setup.